extensions/amp-mowplayer/0.1/amp-mowplayer.js

```diff
@@ -113,9 +129,19 @@ class AmpMowplayer extends AMP.BaseElement {
 
     installVideoManagerForDoc(this.element);
     Services.videoManagerForDoc(this.element).register(this);
+
+    const event = new CustomEvent('mowapiready', {
```

Comment:
Why is this event dispatched at the document level?

Reply:
@alanorozco  It was added in past but is not needed now i think. so i will remove it.